### ISSUE TITLE
Make custom Bazel IntellIJ plugin easy to install for developers, and make sure it can view source code reliably

### ISSUE THREAD

**Issue by lolski** [graknlabs/dependencies#123]
This issue is comprised of two issues:

- the solution should be for the developers to use Max's custom Bazel IntellIJ plugin. We haven't adopted it because the steps to build and install it are a bit hard to follow.
- and even then, if you do use that custom Bazel plugin, there were issues where the "go to source code" functionality doesn't work 100% of the time when viewing the source code of a dependency. This was experienced @adam who tried the plugin last week when getting the simulation up in Grabl

**Comment by lolski** [comment]
We need to prioritise this issue since it has caused productivity dip.

1. design an easy to use distribution for the team
2. get the team to report any bugs as a priority, and provide an updated distribution

**Comment by vmax** [comment]
@lolski my suggestion is to introduce a simple CircleCI pipeline that builds plugin's ZIP and deploys it to our fork's _Releases_ page

**Comment by vmax** [comment]
This is done by publishing a prebuild version of plugin [here](https://github.com/graknlabs/intellij/releases/tag/v1)
